app/src/main/java/com/powerhungers/fomezero/feature/login/data/remote/data_source/LoginRemoteDataSource.kt

```diff
@@ -30,5 +30,7 @@ class LoginRemoteDataSource(private val service: LoginService) {
         cpf = cpf.orEmpty(),
         createdAt = createdAt.orEmpty(),
         updatedAt = updatedAt.orEmpty(),
+        token = token.orEmpty(),
+        refresh_token = refresh_token.orEmpty()
```

Comment:
acredito que alguns desses valores como token nao devem ser armazenados como string vazia, se vc salvar uma string vazia no shared preferences depois tera problemas para re autenticar a sessao do app.
Nesse casso se for vazio ou nulo vc pode jogar uma exception para que o app não prossiga com sucesso ao receber um valor incorreto